README.md

```diff
@@ -2,147 +2,67 @@
-merge the new and/or updated snapshots into the working branch. After this PR is merged, the visual regression tests in
-the working branch should succeed.
-
-Here is the summarised workflow (also described with screenshots in
-[PR #306](https://github.com/silx-kit/h5web/pull/306)):
-
-1. Push your working branch and open a PR.
-2. If the _Cypress_ CI workflow fails, check out the logs.
-3. If the fail is caused by a visual regression (i.e. if a test fails on a `cy.matchImageSnapshot()` call), download the
-   workflow's artifacts.
-4. Review the snapshot diffs:
-
-   - If the visual regressions are unexpected: fix the bug, push it and start from step 2 again.
-   - If the visual regressions are expected: continue to step 5.
-
-5. In the PR, post a comment with the following text: `/approve`.
-6. Go to the _Actions_ page and wait for the _Approve snapshots_ workflow to complete.
-7. Go to the newly opened PR titled _Update Cypress reference snapshots_.
-8. Review the new reference snapshots and merge the PR.
-9. Go back to your main PR and wait for the _Cypress_ workflow to succeed.
-
-### Build
-
-- `npm run build` - build H5Web for production
-- `npm run storybook:build` - build the Component Library's Storybook documentation site
-
-### Code quality
-
-- `npm run lint` - run all linting and code formatting commands
-- `npm run lint:eslint` - lint all TS and JS files with ESLint
-- `npm run lint:tsc` - type-check the whole project, test files included
-- `npm run lint:prettier` - check that all files have been formatted with Prettier
-
-#### Automatic fixing and formatting
-
-- `npm run lint:eslint -- --fix` - auto-fix linting issues
-- `npm run lint:prettier -- --write` - format all files with Prettier
-
-#### Editor integration
+## NPM packages 📚
 
-Most editors support fixing and formatting files automatically on save. The configuration for VSCode is provided out of
-the box, so all you need to do is install the recommended extensions.
+### [@h5web/lib](https://www.npmjs.com/package/@h5web/lib)
 
-### Continuous integration and deployment
+H5Web's component library, which includes the main visualization components
+(`LineVis`, `HeatmapVis`, etc.) as well as some of their lower-level building
+blocks (`VisCanvas`, `ColorBar`, etc.)
 
-- The project's `master` branch is continuously deployed to https://h5web.panosc.eu/ with
-  [Netlify](https://www.netlify.com/).
-- Netlify also deploys each pull request individually to URLs of the form:
-  https:\//deploy-preview-<pr-number\>--h5web.netlify.app/
-- On push to any branch, a [GitHub Action](https://github.com/silx-kit/h5web/actions):
-  - runs the `lint` and `test` scripts;
-  - runs end-to-end tests with Cypress.
-- On push to `master`, another GitHub Action continuously deploys the Component Library's Storybook documentation site
-  to GitHub Pages: https://h5web-docs.panosc.eu
+The library is documented in a Storybook site accessible at
+https://h5web-docs.panosc.eu.
```

Comment:
We should then add the other exported components to the Storybook (`DataCurve`, `HeatmapMesh` ...) ?